### PR TITLE
Added microsoft-office-de

### PR DESCRIPTION
**Pull request by winkelsdorf**
Added Cask for German Installation of Microsoft Office 2011 for Mac

### CODE REVIEW

File: Casks/microsoft-office-de.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  url 'http://officecdn.microsoft.com/pr/MacOffice2011/de-DE/MicrosoftOffice2011.dmg'
+  name 'Microsoft Office German'
+  homepage 'http://www.microsoft.com/mac'
+  license :commercial
+
+  pkg 'Office Installer.pkg'
+
+  uninstall :pkgutil   => 'com.microsoft.office.*',
+            :launchctl => 'com.microsoft.office.licensing.helper'
+  zap       :pkgutil   => [
```
Comment (vitorgalvao) [inline Casks/microsoft-office-de.rb]: Why isn’t this `:pkgutil` in `uninstall`? And won’t it interfere with other Microsoft apps?

Comment (winkelsdorf) [inline Casks/microsoft-office-de.rb]: I don't know, but best to ask your team members, code is taken from official cask repo:
https://github.com/caskroom/homebrew-cask/blob/master/Casks/microsoft-office.rb

Comment (vitorgalvao) [inline Casks/microsoft-office-de.rb]: Let’s get this in now, then, and look at the main one later.

Comment (winkelsdorf) [inline Casks/microsoft-office-de.rb]: Agreed. Not unlikely that you are right about, I just had a quick note at this one as it just required changing the the iso codes for the languages.

**Pull request merged**